Expensify/App
# chore(bump-e-common): Updated expensify-common ref hash

@andreacastejon can you please review this?

### Details
- Merges hyperlink changes from expensify-common to the App

### Fixed Issues

$ Fixes [App/4229](https://github.com/Expensify/App/issues/4229)

### Tests


### QA Steps

1. Type `[Yo (click here to see a cool cat)](https://c8.alamy.com/compes/ha11pc/cookie-cat-con-sombrero-de-cowboy-y-sun-glass-ha11pc.jpg)` (without `)
2. Verify it hyperlinks correctly

### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/3069065/127357607-b09cc0f1-6e6c-4b0d-9d1a-6e53df0e3eec.mp4

#### Mobile Web
https://user-images.githubusercontent.com/3069065/127372250-daec48b7-855b-4965-b11b-7cffddc227cf.mp4

#### Desktop
https://user-images.githubusercontent.com/3069065/127357442-98d02450-cdf1-45d1-8cea-0bf6f136d99b.mp4

#### iOS
https://user-images.githubusercontent.com/3069065/127385748-6e297267-fa5c-47e2-8851-5b0fa654c545.mp4

#### Android
https://user-images.githubusercontent.com/3069065/127372048-5dbce2a9-172b-4252-98d2-6f0945f65ffe.mp4
